packages/git-diff/spec/git-diff-spec.js

```diff
@@ -116,14 +117,15 @@ describe('GitDiff package', () => {
   describe('when a modified file is opened', () => {
     it('highlights the changed lines', () => {
       fs.writeFileSync(
-        path.join(projectPath, 'sample.txt'),
+        path.join(projectPath, 'working-dir', 'sample.txt'),
```

Comment:
This is bad.

Reply:
Damnit I highlighted more lines than that! Where's the container Unit Test in question Github!?